CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 3.4.2 (Under development)
 
+### App Center Crashes
+
+#### iOS
+
+* **[Fix]** Fix sending attachments with a `null` text value.
```

Comment:
Does it now print an error instead of crashing? We don't really send it, maybe the changelog could be more accurate if that's the case.

Reply:
On iOS we can send attachments with null text.